provider/go.mod

```diff
@@ -3,9 +3,9 @@ module github.com/pulumi/pulumi-mailgun/provider/v3
 go 1.19
 
 require (
-	github.com/pulumi/pulumi-terraform-bridge/v3 v3.38.2
-	github.com/pulumi/pulumi/sdk/v3 v3.51.0
-	github.com/wgebis/terraform-provider-mailgun v0.7.4
+	github.com/pulumi/pulumi-terraform-bridge/v3 v3.40.0
```

Comment:
I'm excited one day for these to happen automatically in their own PRs.. there's fewer things then that's changing in one PR which is always nicer to review.